settings/zynq7.sh

```diff
@@ -1,5 +1,5 @@
 export XRAY_DATABASE="zynq7"
-export XRAY_PART="xc7z010clg400-1"
+export XRAY_PART="xc7z020clg400-1"
```

Comment:
I don't think we want to only output tile grid for the xc7z020clg400.  In the long run we need tilegrid for multiple classes of parts.  Do you want to propose something akin to https://github.com/SymbiFlow/prjxray/blob/master/Makefile#L165 allow for tilegrids of multiple parts within the same family?

Reply:
Sure, we definitely need support per part. This WIP PR is currently for testing how well suited are our fuzzers currently for 7020 parts.